### DISCUSSION TITLE
Stucked actions from Dependabot

### DISCUSSION THREAD

**Discussion by sanfair** [dependabot/dependabot-core#10240] [Q&A]
Is there a way to cancel Dependabot actions that are stuck in the queue?

I don't have access to cancel them (red bar on screenshot)

![image](https://github.com/user-attachments/assets/fb446b74-7c8d-4e05-9ee7-f3e395a740a2)

I think it happened after temporarily disabling GitHub Actions while merging Dependabot PRs 
(I don't want to trigger all GHA for each of the 20+ PR merges from Dependabot, so I usually turn off actions while merging these PRs and turn them on before merging the last one)

There were also corresponding Dependabot runs in `Insights > Dependency graph > Dependabot,` but it seems they were completed by an additional GHA created about an hour ago.

**Comment by abdulapopoola** [comment]
@sanfair , this should be fixed now. Does it still happen?

**Reply by sanfair** [reply]
All queued jobs that were stuck are gone now.
I will check next week if it happen again after disabling GitHub actions 
Thank you

**Comment by MatthewSteeples** [comment]
I'm experiencing a similar problem to this one. I re-queued a failed Dependabot Action and now it's just stuck in the Queued state

**Reply by netflash** [reply]
same here, in an enterprise saas emu org.
How can I resolved it?
